.github/workflows/pr-comment.yaml

```diff
@@ -54,9 +54,20 @@ jobs:
           path: pr/benchmark
           if_no_artifact_found: ignore
 
+      - uses: dawidd6/action-download-artifact@v2
```

Comment:
The ecosystem and benchmark jobs use the comment action because they don't automatically fail when there's a regression. The assessment is left to the user. Using a comment also has the downside that it creates noise because GitHub sends a notification every time the comment is created or updated. 

The fact the udeps step has a clear failure/pass condition and the added noise of the comment are the reasons why I would prefer to keep the job similar to the clippy/test/fmt jobs, where it fails (this also generates a notification depending on your settings) without creating/updating an additional comment.